src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <value>Create Reports</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Standard Report Type in GS for End User</shortDescription>
+        <value>Create a Report from a Standard Report Type</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item View Dashboard in GS for End User</shortDescription>
+        <value>Dashboards pull data from one of more reports and display it side by side in the same window. NPSP includes three dashboards that you can use as is or customize.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item View Dashboard in GS for End User</shortDescription>
+        <value>Learn more about Dashboards in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item View Dashboard in GS for End User</shortDescription>
+        <value>View Dashboards</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item View Dashboard in GS for End User</shortDescription>
+        <value>View a Dashboard</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistYourDataDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Your Data in GS for End User</shortDescription>
+        <value>Reports and Dashboards help organize your data so that you can tell the story of your organization&apos;s impact. NPSP includes out-of-the-box reports, or you can create your own.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistYourDataTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Your Data in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Title of Understand Your Data checklist item in Get Started for End Users.